wrapanapi/systems/msazure.py

```diff
@@ -672,6 +672,13 @@ def list_security_group(self):
                 sec_groups_in_location.append(sec_gp.name)
         return sec_groups_in_location
 
+    def list_security_group_ports(self, resource_group, sec_group_name):
```

Comment:
With other AzureSystem class methods that take resource group, its optional and defaults to the instance attribute (`self.resource_group`). Please follow that pattern here, making resource_group an optional param.